src/simplifiedSummaryBlock/sass/simplified-summary-block.scss

```
@@ -0,0 +1,9 @@
+// Editor-only styles for the simplified summary block placeholder.
+.edac-simplified-summary {
+	.edac-simplified-summary-block__placeholder {
+		border: 1px dashed currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)